.env

```diff
@@ -1 +1,3 @@
-JWT_KEY=teamwork
\ No newline at end of file
+JWT_KEY=teamwork
+
+DATABASE_URL=postgres://postgres:postgres@localhost:5432/teamworkdb
```

Comment:
The .env file shouldn't available to the public because there's no need to have it if it will be available. Git-ignore this file.